src/model/log_prob.jl

```diff
@@ -3,8 +3,6 @@
 using Distributions
 import ..SensitiveFloats: SensitiveFloat, zero_sensitive_float, clear!
 
-#include("../bivariate_normals.jl")
-#include("log_prob_util.jl")
 EPS = 1e-6
```

Comment:
EPS should be maked `const`...this can make a big different for runtime.

I haven't seen the all caps convention used in Julia for constants.

Also, `eps` could be confused with `espilon_mat` in the Celeste model. Is there another way to do this?